plugins/GrapesJsBuilderBundle/EventSubscriber/SerializerSubscriber.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace MauticPlugin\GrapesJsBuilderBundle\EventSubscriber;
+
+use JMS\Serializer\EventDispatcher\Events;
+use JMS\Serializer\EventDispatcher\EventSubscriberInterface;
+use JMS\Serializer\EventDispatcher\ObjectEvent;
+use JMS\Serializer\JsonSerializationVisitor;
+use JMS\Serializer\Metadata\StaticPropertyMetadata;
+use Mautic\EmailBundle\Entity\Email;
+use Mautic\InstallBundle\Install\InstallService;
+use MauticPlugin\GrapesJsBuilderBundle\Model\GrapesJsBuilderModel;
+
+class SerializerSubscriber implements EventSubscriberInterface
+{
+    public function __construct(
+        private GrapesJsBuilderModel $grapesJsBuilderModel,
+        private InstallService $installer,
+    ) {
+    }
+
+    /**
+     * @return array<int, array<string, string>>
+     */
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            [
+                'event'  => Events::POST_SERIALIZE,
+                'method' => 'addCustomMJML',
+            ],
+        ];
+    }
+
+    public function addCustomMJML(ObjectEvent $event): void
+    {
+        if (!$this->installer->checkIfInstalled()) {
+            return;
+        }
```

Comment:
Aside from checking mautic is installed, we should also check to only send this data when grapejsbuilder module is 'enabled'. See https://github.com/mautic/mautic/blob/5.x/plugins/GrapesJsBuilderBundle/EventSubscriber/AssetsSubscriber.php
To also add a `if ($this->config->isPublished()) {` check.

Reply:
Not wanting to be annoying, but probably it's more clear to do. 
`if ('$this->config->isPublished()) {
            return;
}`
Before we start doing something, to make it a bit more readable.

Reply:
No worries. I have update the PR.

Reply:
Looks good, you can mark this as resolved.